public/tier1/convar.h

```diff
@@ -49,21 +50,39 @@ class ConCommand;
 class ConCommandBase;
 class ConVarRefAbstract;
 class ConCommandRefAbstract;
-struct characterset_t;
 
 class ALIGN8 ConVarHandle
 {
 public:
-    bool IsValid() { return value != kInvalidConVarHandle; }
-	uint32 Get() { return value; }
-	void Set( uint32 _value ) { value = _value; }
+	ConVarHandle() { m_value = 0xFFFFFFFF; };
 
-private:
-    uint32 value = kInvalidConVarHandle;
+	bool IsValid() const;
+	void Invalidate();
 
 private:
-	static const uint32 kInvalidConVarHandle = 0xFFFFFFFF;
+	friend bool operator!=(const ConVarHandle& lhs, const ConVarHandle& rhs);
+	friend bool operator==(const ConVarHandle& lhs, const ConVarHandle& rhs);

```

Comment:
In the SDK code style '(' and ')' come with extra spaces, & and * are placed next to variable name.

Reply:
Coding style is widly inconsistent across the SDK. We also have other parts of this file with the very same coding style I applied.

I'll apply the spacing comment for parenthesis though